gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright © 2015-2021 the original authors.
+# Copyright � 2015-2021 the original authors.
```

Comment:
Was there a small hickup here perhaps? I would not expect these changes here.

Reply:
This was auto generated after running the command. I did not touch these files manually